tensorflow/python/kernel_tests/matmul_op_test.py

```diff
@@ -441,5 +442,47 @@ def testTransposedStatistics(self):
           self.assertEqual(7200, flops)
 
 
+# @ operator supported since python 3.5.
+if sys.version_info >= (3, 5):
+  # Use eval() to avoid a compilation error on earlier versions.
+  _MATMUL = lambda x, y: eval("x @ y")
+else:
```

Comment:
Lines 460 to 466 are unused for now, but I think it's better to keep them here just in case someone adds more tests: they make _MATMUL work very much the same way whatever the version of python used to run the tests.